src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx

```diff
@@ -406,6 +406,9 @@ function ComposerWithSuggestions(
      */
     const updateComment = useCallback(
         (commentValue: string, shouldDebounceSaveComment?: boolean) => {
+            if (isIOSNative) {
+                syncSelectionWithOnChangeTextRef.current = null;
+            }
```

Comment:
Or should we remove https://github.com/Expensify/App/blob/2514f2988eb26858d30fa501c0040b1f8fc2e239/src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.tsx#L442-L444 if you set it to null above

Reply:
If we remove it, https://github.com/Expensify/App/issues/29405 will happen again.